app/components/experience.tsx

```diff
@@ -2,6 +2,7 @@ import { TicketContainer } from "@/components/ticket-container";
 
```

Comment:
This is admittedly a TODO in all but name, but also, it's true - too much repetition here. This is just a fix for now.